backend/scripts/nlp/nltk-json.ipynb

```diff
@@ -0,0 +1,320 @@
+    "\n",
+    "    with open('emotions.txt', 'r') as file:\n",
+    "        for line in file:\n",
+    "            clearLine = line.replace('\\n', '').replace(\n",
+    "                ',', '').replace(\"'\", '').replace(' ', '').strip()\n",
+    "            word, emotion = clearLine.split(':')\n",
+    "\n",
+    "            if word in finalList:\n",
+    "                emotionList.append(emotion)\n",
+    "                wordList.append(word)\n",
+    "            countEmotions = Counter(emotionList)\n",
+    "            countwords = Counter(finalList)\n",
+    "    totalCount = count+len(set(tokenizedList) -\n",
+    "                           set(wordList)-set(stopwords.words('english')))\n",
+    "\n",
+    "    # myFile =Path('/college/Winter2022/Ad450/NLP/test.myFile').read_text().replace('\\n','')\n",
+    "\n",
+    "# creating a blob text\n",
+    "\n",
+    "    blob = TextBlob(myFile)\n",
+    "    wordCount = blob.word_counts\n",
+    "\n",
+    "# make all the typos correct\n",
+    "\n",
+    "    # correctmyFile = blob.correct()\n",
+    "    # sentences = blob.sentences\n",
+    "    # sen = blob.sentiment\n",
+    "\n",
+    "    blob = TextBlob(myFile, analyzer=NaiveBayesAnalyzer())\n",
+    "    sen_sub = blob.sentiment\n",
+    "\n",
+    "    def sentimentAnalyse(text):\n",
+    "        score = SentimentIntensityAnalyzer().polarity_scores(text)\n",
+    "        for k,v in score.items():\n",
+    "            print(f\"{k}:{v:.2f}\")\n",
+    "        neg = score['neg']\n",
+    "        pos = score['pos']\n",
+    "        if neg > pos:\n",
+    "            print(\"\\n\"+\"In general: Negative sentiment!\")\n",
+    "        elif pos > neg:\n",
+    "            print(\"\\n\"+\"In general: Positive sentiment!\")\n",
+    "        else:\n",
+    "            print(\"\\n\"+\"In general: Neutral vibe!\")\n",
+    "\n",
+    "    sentimentAnalyse(myFile)\n",
+    "    reg = RegexpTokenizer('(?u)\\w+|\\$[\\d\\.]+|\\s+')\n",
+    "    # regex_tokenize = RegexpTokenizer.tokenize(myFile)\n",
+    "    # print(regex_tokenize)\n",
+    "\n",
+    "    print('\\n'+'******************************************************************')\n",
+    "    print('Summary of sentiments:')\n",
+    "    print('\\n'+'list of words with emotions: ', wordList)\n",
+    "    print('Summary of emotions count: ', countEmotions)\n",
+    "    print('\\n'+'******************************************************************')\n",
+    "    print('                      End of Application                          ')\n",
+    "    print('******************************************************************')\n",
+    "\n",
+    "# Drawing the bar chart\n",
+    "\n",
+    "    fig, axl = plt.subplots()\n",
+    "    axl.bar(countEmotions.keys(), countEmotions.values())\n",
+    "    fig.autofmt_xdate()\n",
+    "    plt.savefig('graph.png')\n",
+    "    plt.show()\n",
+    "\n",
+    "    print(nltk.pos_tag(tokenizedList))\n",
```

Comment:
Not certain the value of including lines 265-267. They appear to print the data to the console and runner a grammar check, but it's at the end of the script.